docs/multi-echo.rst

```diff
@@ -54,6 +54,31 @@ We can use this information to denoise the optimally combined time series.
 .. _processing pipeline details: https://tedana.readthedocs.io/en/latest/approach.html#optimal-combination
 .. _Pruim et al. (2015): https://www.sciencedirect.com/science/article/pii/S1053811915001822
 
+Recommendations on multi-echo use for someone planning a new study
+------------------------------------------------------------------
+Multi-echo fMRI acquisition sequences and analysis methods are rapidly maturing. Someone who has access
+to a multi-echo fMRI sequence should seriously consider using it. Multiple studies have shown that a
+weighted average of the echoes to optimize T2* weighting, sometimes called "optimally combined,"
+gives a reliable, modest boost in data quality. The optimal combination of echoes can currently be
+calculated in several software packages including AFNI, fMRIPrep, and tedana. In tedana, the weighted
+average can be calculated with https://tedana.readthedocs.io/en/latest/usage.html#run-t2smap If no other
```

Comment:
```suggestion
average can be calculated with t2smap_ If no other
```
Paired with a below suggestion would make the link in-line, which would improve readability slightly.

Reply:
At the end, with the other links at the bottom of the file, add
` .. _t2smap: https://tedana.readthedocs.io/en/latest/usage.html#run-t2smap